docs/rst/use_cases/ros_cloud.rst

```diff
@@ -0,0 +1,144 @@
+.. include:: ../exports/alias.include
+
+.. _ros_cloud:
+
+####################
+ROS 2 and Kubernetes
+####################
+
+Apart from plain LAN-to-LAN communication, Cloud environments such as container-oriented platforms have also been
+present throughout the |ddsrouter| design phase. In this walk-through example, we will set up both a *Kubernetes*
+(|k8s|) network and local environment in order to establish communication between a pair of ROS nodes, one sending
+messages from a LAN (talker) and another one (listener) receiving them in the Cloud. This will be accomplished by having
+a |ddsrouter| instance at each side of the communication.
+
+.. figure:: /rst/figures/ddsrouter_overview_wan.png
+
+Local setup
+===========
+The local instance of |ddsrouter| (local router) only requires to have a
+:ref:`Simple Participant <user_manual_participants_simple>`, and a :ref:`WAN Participant <user_manual_participants_wan>`
+that will play the client role in the discovery process of remote participants
+(see :term:`Discovery Server discovery mechanism <Discovery Server>`).
+
+After having acknowledged each other’s existence through `Simple DDS discovery mechanism <https://fast-dds.docs.eprosima.com/en/latest/fastdds/discovery/simple.html>`_
+(multicast communication), the local participant will start receiving messages published by the ROS 2 talker node, and
+will then forward them to the WAN participant. Following, these messages will be sent to another participant hosted on a
+|k8s| cluster to which it connects via WAN communication over UDP/IP.
+
+Following is a representation of the above-described scenario:
+
+.. figure:: /rst/figures/vulcanexus_local.png
+
+
+Local router
+------------
+The configuration file used by the local router will be the following:
+
+.. literalinclude:: ../../resources/use_cases/ros_cloud/local-ddsrouter.yaml
+    :language: yaml
+
+Note that the simple participant will be receiving messages sent in DDS domain ``0``. Also note that, due to the choice
+of UDP as transport protocol, a listening address with the LAN public IP address needs to be specified for the local WAN
+participant, even when behaving as client in the participant discovery process. Make sure that the given port is
+reachable from outside this local network by properly configuring port forwarding in your Internet router device.
+The connection address points to the remote WAN participant deployed in the |k8s| cluster. For further details on how to
+configure WAN communication, please have a look at :ref:`WAN Configuration <user_manual_wan_configuration>`.
+
+To launch the local router, execute:
+
+.. code-block:: bash
+
+    ddsrouter -c local-ddsrouter.yaml
```

Comment:
NIT:
In general, try to use the long argument name (unless you have named it before in the page)
```suggestion
    ddsrouter --config-path local-ddsrouter.yaml
```

Reply:
:+1: